rosie/chamber_of_deputies/tests/test_adapter.py

```diff
@@ -27,6 +30,18 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self.temp_path)
 
+    def test_quantity_of_downloaded_datasets_all_years_fetch(self):
+        years = AVAILABLE_YEARS
+        chamber_of_deputies = Dataset(self.temp_path, years)
+        datasets = chamber_of_deputies.fetch()
+        self.assertEqual(10, len(datasets))
```

Comment:
This hard-coded `10` will break tests every new year. `len(AVAILABLE_YEARS)` wont ; )

Reply:
accepted, I just didn't know we could do that